source/vir/src/ast_util.rs

```diff
@@ -57,6 +58,16 @@ pub fn types_equal(typ1: &Typ, typ2: &Typ) -> bool {
     }
 }
 
+pub fn bitwidth_from_type(et: &Typ) -> Option<u32> {
+    if let TypX::Int(IntRange::U(size)) = &**et {
+        return Some(*size);
+    }
+    if let TypX::Int(IntRange::I(size)) = &**et {
```

Comment:
We should probably not have this case, since we don't yet have support for singed  bitvector.